File: app/src/main/java/com/main/climbingdiary/database/entities/Route.java
package com.main.climbingdiary.database.entities;

import org.chalup.microorm.annotations.Column;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class Route implements RouteElement{
    @Column("id") private int id;
    @Column("stil") private String style;
    @Column("level") private String level;
    @Column("name") private String name;
    @Column("sektor") private String sector;
    @Column("gebiet") private String area;
    @Column("rating") private int rating;
    @Column("kommentar") private String comment;
    @Column("date") private String date;
}

File: app/src/main/java/com/main/climbingdiary/database/entities/Route.kt
package com.main.climbingdiary.database.entities

import com.main.climbingdiary.models.Styles
import org.chalup.microorm.annotations.Column

data class Route(
    @Column("id") var id: Int = 0,
    @Column("stil") var style: String = Styles.getRP(),
    @Column("level") override var level: String = "8a",
    @Column("name") override var name: String? = null,
    @Column("sektor") override var sector: String? = null,
    @Column("gebiet") override var area: String? = null,
    @Column("rating") var rating: Int? = null,
    @Column("kommentar") var comment: String? = null,
    @Column("date") var date: String? = null,
    @Column("tries") var tries: Int? = null
) : RouteElement